website/docs/layout.md

```diff
@@ -100,6 +100,10 @@ q.page['tab_card'] = ui.tab_card(
 
 ![zone card shrink](assets/zone-card-shrink.png)
 
+### Setting Width and Height
+
+By default, we encourage static units like `px` or `rem` for setting widths and heights. In flex layouts, percent values such as `100%` only work if every ancestor has an explicit size; otherwise, the computed height can be zero and the content may not render as expected.
```

Comment:
Would be good to also include an MDN link to all the available units.
```suggestion
By default, we encourage static units like `px` or `rem` for setting widths and heights. In flex layouts, percent values such as `100%` only work if every ancestor has an explicit `size`, otherwise, the computed height can be zero and the content may not render as expected.
```